src/components/network/network-panel.html

```diff
@@ -19,8 +19,14 @@
   </div>
   <div id="combined-regulation" class="panel-section">
     <div class="item-title">Combined Regulations</div>
-    <button class="btn btn-default glyphicon glyphicon-refresh btn-update"></button>
-    <input class="form-control input-with-update">
+    <input class="form-control">
+    <div id="combined-input" class="input-checkbox">
+      <label class="radio-inline" name="string"><input type="radio" name="regulator-optradio" checked>text</label>
+      <label class="radio-inline" name="regex"><input type="radio" name="regulator-optradio">regex</label>
+    </div>
+    <div class="sub-buttons">
+      <button id="refresh-combined" class="btn btn-default btn-xs glyphicon glyphicon-refresh" title="refresh genes"></button>
```

Comment:
id="refresh" is sufficient as the parent container already have "combined-regulation" as its id that indicates the purpose of this entire block.